docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_6.md

```diff
@@ -0,0 +1,113 @@
+## ExperimentalDetectronGenerateProposalsSingleImage <a name="ExperimentalDetectronGenerateProposalsSingleImage"></a> {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_6}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronGenerateProposalsSingleImage* computes ROIs and their scores 
+based on input data.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Transposes and reshape predicted bounding boxes deltas and scores to get them into the same order as the anchors;
+2.  Transforms anchors into proposals using deltas and clips proposals to image;
+3.  Removes predicted boxes with either height or width < *min_size*;
+4.  Sorts all `(proposal, score)` pairs by score from highest to lowest, order of pairs with equal scores is undefined;
+5.  Takes top *pre_nms_count* proposals, if total number of proposals is less than *pre_nms_count* then operation takes 
+all proposals;
+6.  Applies non-maximum suppression with *nms_threshold*;
+7.  Takes top *post_nms_count* proposals and return these top proposals and their scores. If total number of proposals 
+is less than *post_nms_count* then operation returns output tensors filled by zeroes.
```

Comment:
the rest of the output is filled by zeros

Reply:
No, in this case operation returns two tensors filled by zeroes, see code below in operation realization: https://github.com/openvinotoolkit/openvino/blob/83e5bde4ea9ee7d91b852ecc9707b78b01fcc7f1/inference-engine/src/mkldnn_plugin/nodes/proposal_onnx.cpp#L264